tueplots/figsizes.py

```diff
@@ -169,28 +177,40 @@ def neurips2021(
     )
 
 
-def _from_base_pt(*, width_pt, **kwargs):
-    width_in = width_pt / _POINTS_PER_INCH
-    return _from_base_in(width_in=width_in, **kwargs)
+def _from_base_pt(*, base_width_pt, **kwargs):
+    base_width_in = base_width_pt / _POINTS_PER_INCH
+    return _from_base_in(base_width_in=base_width_in, **kwargs)
 
 
-def _from_base_in(*, width_in, nrows, height_to_width_ratio, ncols):
-    height_in = height_to_width_ratio * width_in * nrows / ncols
+def _from_base_in(*, base_width_in, rel_width=1.0, height_to_width_ratio, nrows, ncols):
```

Comment:
do you think it would be more transparent if rel_width was not a default at 1.0, but instead 1.0 would be passed from the outside wherever it is explicitly ignored (like in the half-column functions)?

Reply:
Yes, I agree, it would be more transparent 👍🏻 I initially decided against it to keep the diff small, but I guess that does not matter at this point 😁